server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/QualityGatesWsSupport.java

```diff
@@ -56,19 +56,32 @@ QualityGateConditionDto getCondition(DbSession dbSession, long id) {
     return checkFound(dbClient.gateConditionDao().selectById(id, dbSession), "No quality gate condition with id '%d'", id);
   }
 
+  /**
+   * @deprecated use {@link #isQualityGateAdmin(OrganizationDto)} instead
+   */
+  @Deprecated
   boolean isQualityGateAdmin() {
     return userSession.hasPermission(ADMINISTER_QUALITY_GATES, defaultOrganizationProvider.get().getUuid());
   }
 
+  boolean isQualityGateAdmin(OrganizationDto organization) {
+    return userSession.hasPermission(ADMINISTER_QUALITY_GATES, organization);
+  }
+
   WebService.NewParam createOrganizationParam(NewAction action) {
     return action
       .createParam(PARAM_ORGANIZATION)
       .setDescription("Organization key. If no organization is provided, the default organization is used.")
+      .setSince("7.0")
       .setRequired(false)
       .setInternal(false)
       .setExampleValue("my-org");
   }
 
+  /**
+   * @deprecated use {@link #getActions(OrganizationDto, QualityGateDto, QualityGateDto)} instead
+   */
+  @Deprecated
   Qualitygates.Actions getActions(QualityGateDto qualityGate, @Nullable QualityGateDto defaultQualityGate) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)